modules/accessing-s3-compatible-object-storage-with-self-signed-certificates.adoc

```diff
@@ -0,0 +1,80 @@
+:_module-type: PROCEDURE
+
+[id='accessing-s3-compatible-object-storage-with-self-signed-certificates_{context}']
+= Accessing S3-compatible object storage with self-signed certificates
+
+[role='_abstract']
+To use object storage solutions or databases that are deployed in an OpenShift cluster that uses self-signed certificates, you need to configure {productname-short} to trust the cluster's certificate authority (CA).
+
+Each namespace has a ConfigMap called `kube-root-ca.crt` that contains the CA certificates of the internal API Server. Use the following steps to configure {productname-short} to trust the certificates issued by `kube-root-ca.crt`.
+
+ifdef::upstream[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{odhdocshome}/installing-open-data-hub/#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+ifndef::upstream[]
+
+ifdef::self-managed[]
+
+ifndef::disconnected[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+ifdef::disconnected[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+endif::[]
+
+ifdef::cloud-service[]
+Alternatively, you can add a custom CA bundle by using the OpenShift console, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs#adding-a-ca-bundle_certs[Adding a CA bundle].
+endif::[]
+
+endif::[]
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+* You have an object storage solution or database deployed in your OpenShift cluster.
```

Comment:
To provide the user with more information on the prereq specified on line 38, is there a suitable module to link to for configuring an object storage solution or database deployed in an OpenShift cluster?